elm/lib/dependabot/elm/update_checker/requirements_updater.rb

```diff
@@ -53,19 +64,23 @@ def update_requirement(old_req, new_version)
           end
         end
 
+        sig { params(minimum: T.untyped, version: T.untyped).returns(String) }
```

Comment:
Looks like minimum is String / Integer and version is String / Integer / Version but I'll leave you to decide.